x-pack/test/functional/apps/dashboard_mode/dashboard_view_mode.js

```diff
@@ -55,6 +57,12 @@ export default function ({ getService, getPageObjects }) {
       await PageObjects.dashboard.saveDashboard(dashboardName);
     });
 
+    after(async () => {
+      await kibanaServer.importExport.unload(
+        'x-pack/test/functional/fixtures/kbn_archiver/dashboard_view_mode'
+      );
```

Comment:
I don't think this `unload` is actually enough in this test.  This test file saves a new saved search and a new dashboard.  I think we would want to clean those with something like `.clean(['search', 'dashboard'[);`

Reply:
@LeeDr Looks like the test user svc is already restoring defaults.  Is there more to be done regarding test users?

Reply:
It's not about the test user, it's about the saved objects.  Currently the test does this;
1. load some saved objects from a kbnArchive
2. save some new SOs as a part of the test
3. unload the SOs from step 1.  This leaves the SOs that were created in step 2.  The test should clean those up (delete the SOs).

Reply:
I think the latest push addresses this boss.